app/views/activity_insight_oa_workflow/wrong_file_version_curation/index.html.erb

```diff
@@ -0,0 +1,69 @@
+<div class="container pt-4 pb-5">
+  <%= link_to '<< Back', activity_insight_oa_workflow_path %>
+  <h1 class="pb-3 pt-2">Publications with Wrong File Version</h1>
+  <div class='card'>
```

Comment:
I removed this `.card` container in all of the other list views to allow the larger tables to take up the full width of the screen. We'll probably want to remove it here as well.